extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -108,6 +110,7 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 				slotTargetingData.src = 'rec';
 			}
 
+			slotTargetingData.passback = passbackHandler.get(slotName) || 'none';
```

Comment:
in AC of the ticket we have `default: passback=unknown` and here we have `none` - which one is correct?